roles/beaker/sandbox/packages/tasks/main.yml

```diff
@@ -9,42 +9,41 @@
 
 - name: install runtime dependencies
   package:
-    name: "{{ item }}"
+    name:
+        - python-gunicorn
+        - nodejs-less
+        - syslinux-tftpboot
+        - libvirt-client
+        - firefox # for tests
+        - openldap-servers
+        - python-gunicorn
```

Comment:
We need one more
**python-gssapi**.
It is necessary for common

Reply:
there it is